Assets/Plugins/Source/Core/PlatformManager.cs

```diff
@@ -99,27 +99,40 @@ public static Platform CurrentPlatform
 
         static PlatformManager()
         {
-            AddPlatformInfo(Platform.Android, "Android", "eos_android_config.json", typeof(AndroidConfig), ".so");
-            AddPlatformInfo(Platform.iOS, "iOS", "eos_ios_config.json", typeof(IOSConfig), ".dylib");
-            AddPlatformInfo(Platform.Linux, "Linux", "eos_linux_config.json", typeof(LinuxConfig), ".so");
-            AddPlatformInfo(Platform.macOS, "macOS", "eos_macos_config.json", typeof(MacOSConfig), ".dylib");
-            AddPlatformInfo(Platform.Steam, "Steam", "eos_steam_config.json", typeof(EOSSteamConfig), "");
+            AddPlatformInfo(Platform.Android,     "Android",       "eos_android_config.json"    );
+            AddPlatformInfo(Platform.iOS,         "iOS",           "eos_ios_config.json"        );
+            AddPlatformInfo(Platform.Linux,       "Linux",         "eos_linux_config.json"      );
+            AddPlatformInfo(Platform.macOS,       "macOS",         "eos_macos_config.json"      );
+            AddPlatformInfo(Platform.Steam,       "Steam",         "eos_steam_config.json"      );
+            AddPlatformInfo(Platform.XboxOne,     "Xbox One",      "eos_xb1_config.json"        );
+            AddPlatformInfo(Platform.XboxSeriesX, "Xbox Series X", "eos_xboxseriesx_config.json");
```

Comment:
This config file name is incorrect: it should be `eos_xsx_config.json`. Changing the name of the config file would be a breaking change, i.e. can't be in a hotfix.